tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Mention what people should be looking for i.e. "rerun this cell periodically until the task 'state' reads 'COMPLETED'"</li><li>I think add something like "Side note: the task will also be registered in the JavaScript Code Editor's list of tasks."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Are you sure the status should reads 'COMPLETED' and not 'READY' ?</p><p> </p><p>I modified as follow:</p><p> </p><blockquote>Then we can check the status of our task (note: the task will also be registered in the JavaScript&nbsp;<a href="https://code.earthengine.google.com/" target="_blank">Code Editor's</a>&nbsp;list of tasks) using the status() method. Depending on the size of the request we might run this cell several times until the task&nbsp;<em>state</em>&nbsp;becommes 'READY'.</blockquote>

Reply:
I checked again and the order I got as I repeated running the cell was: READY, RUNNING, and COMPLETE.

Reply:
<p>I checked again and the order I got as I repeated running the cell was: READY, RUNNING, and COMPLETE.</p>